service/src/main/java/greencity/service/EventServiceImpl.java

```diff
@@ -394,19 +396,16 @@ private void addNewImages(Event toUpdate, UpdateEventDto updateEventDto, Multipa
         }
     }
 
-    private void addAddressesToLocation(List<EventDateLocationDto> eventDateLocationDtos) {
-        for (var date : eventDateLocationDtos) {
-            if (date.getCoordinates() != null) {
-                CoordinatesDto coordinatesDto = date.getCoordinates();
-                List<GeocodingResult> address = googleApiService.getResultFromGeoCodeByCoordinates(
-                    new LatLng(coordinatesDto.getLatitude(), coordinatesDto.getLongitude()));
-                GeocodingResult resultUa = address.get(0);
-                GeocodingResult resultEn = address.get(1);
-                coordinatesDto.setAddressUa(resultUa.formattedAddress);
-                coordinatesDto.setAddressEn(resultEn.formattedAddress);
-                date.setCoordinates(coordinatesDto);
-            }
-        }
+    private void addAddressToLocation(List<EventDateLocationDto> eventDateLocationDtos) {
+        eventDateLocationDtos
+            .stream()
```

Comment:
`googleApiService` does 2 calls for every `event` so it could take a lot of time and our client could have `timeout exception` 
Maybe it is better to do it in parallel?